sapl/materia/views.py

```diff
@@ -2072,6 +2072,11 @@ def get_context_data(self, **kwargs):
         qr = self.request.GET.copy()
         context['object_list'] = context['object_list'].order_by(
             'ano', 'numero')
+        not_list = [self.kwargs['pk']]
+        principal = MateriaLegislativa.objects.get(pk=self.kwargs['pk'])
+        for m in principal.materia_principal_set.all():
```

Comment:
Daria pra fazer assim:
`not_list = [self.kwargs['pk']] + [m.materia_anexada.id for m in principal.materia_principal_set.all()]`